documentation/source/users/rmg/installation/ubuntuSubsystem.rst

```diff
@@ -0,0 +1,43 @@
+.. _ubuntuSubsystem:
```

Comment:
Why do we call it Ubuntu Subsystem? The official name is "Windows Subsystem for Linux". I think that "Linux Subsystem" is a good short name

Reply:
I'll make this change, thanks for the suggestion!